pyspider/database/elasticsearch/projectdb.py

```diff
@@ -40,15 +42,15 @@ def insert(self, name, obj={}):
         obj.setdefault('rate', 0)
         obj.setdefault('burst', 0)
 
-        return self.es.index(index=self.index, doc_type=self.__type__, body=obj, id=name,
-                             refresh=True)
+        return self.es.index(index=self.index, doc_type=self.__type__, body=obj, id=name)
 
-    def update(self, name, obj={}, **kwargs):
-        obj = dict(obj)
+    def update(self, name, obj: Dict = None, **kwargs):
+        if obj is None:
+            obj = dict()
         obj.update(kwargs)
         obj['updatetime'] = time.time()
         return self.es.update(index=self.index, doc_type=self.__type__,
-                              body={'doc': obj}, id=name, refresh=True, ignore=404)
```

Comment:
Same here, refresh is to make sure the change to project are visible immediately. The updates on projectdb is expected to be low, that while used as database, it's exepcted for user to see the change immediately.